src/Packetery/Module/Options/Provider.php

```diff
@@ -179,32 +179,32 @@ public function replaceShippingAddressWithPickupPointAddress(): bool {
 	public function getLabelFormats(): array {
 		return [
 			'A6 on A4'       => [
-				'name'         => __( 'labelNameA6onA4', 'packetery' ),
+				'name'         => __( '1/4 A4, print on A4, 4pcs/page', 'packetery' ),
```

Comment:
již nepoužívané klíče k překladu,  je potřeba odstranit také ze po a pot souborů.

Reply:
Pokud to bude udělané v každé větvy samostatně tak tam budou zase vznikat konflikty kvůli tomuto. Lepší by bylo všechny takové překlady přesunout do PHP a po/pot soubory navždy smazat.

Reply:
Souhlasím,  dropni tedy odstranění těch klíčů z po a mo souborů.  Překladové soubory odstraníme v rámci jiného ticketu.

Reply:
Překladové soubory jsem aktualizoval v posledním komitu